app/uk/gov/hmrc/tai/viewModels/employments/UpdateEmploymentCheckYourAnswersViewModel.scala

```diff
@@ -48,7 +48,7 @@ case class UpdateEmploymentCheckYourAnswersViewModel(id: Int,
       controllers.employments.routes.UpdateEmploymentController.addTelephoneNumber().url
     )
 
-    if (contactByPhone.equals("Yes")) {
+    if (contactByPhone == Messages("tai.label.yes")) {
```

Comment:
If we have toggled welsh on, would the contents of contactByPhone will be checked against the Welsh version of tai.label.yes?  If(contactByPhone == Iwan)?

Reply:
Indeed, that has been tested